common/static/common/js/spec/components/paginated_view_spec.js

```diff
@@ -0,0 +1,182 @@
+define([
+    'backbone',
+    'underscore',
+    'common/js/spec_helpers/ajax_helpers',
+    'common/js/components/views/paginated_view',
+    'common/js/components/collections/paging_collection'
+], function (Backbone, _, AjaxHelpers, PaginatedView, PagingCollection) {
+    'use strict';
+    describe('PaginatedView', function () {
+        var TestCardView = Backbone.View.extend({
```

Comment:
@andy-armstrong - I think you're picking up on something which I actually already fixed. Unfortunately I just forgot to rename this variable. I'll take care of that.